### ISSUE TITLE
[bug] When switch between tabs, there is a blank space at the top if use Tabs.ScrollView

### ISSUE THREAD

**Issue by devoren** [PedroBern/react-native-collapsible-tab-view#354]
Hey 👋 When I switch between tabs, there is a blank space at the top if I use Tabs.ScrollView:
` "react-native-collapsible-tab-view": "^6.1.4"`

Code: 
```
const tabs = useMemo(
() => [
	{ key: 'about', title: 'T' },
	{ key: 'products', title: 'A' },
	{ key: 'categories', title: 'S' },
	{ key: 'reviews', title: 'F' },
],
[],
);

const renderHeader = useCallback(() => {
return <Header title={storeName} />;
}, []);

return (
<View style={{ flex: 1, backgroundColor: colors.gray100 }}>
	<Tabs.Container
		headerHeight={undefined}
		renderTabBar={(props) => (
			<MaterialTabBar
				{...props}
				activeColor={colors.primary}
				inactiveColor={colors.gray900}
				labelStyle={styles.labelStyle}
				tabStyle={styles.tabStyle}
				scrollEnabled
				indicatorStyle={styles.indicatorStyle}
				contentContainerStyle={{
					gap: scale.sx,
				}}
			/>
		)}
		renderHeader={renderHeader}
		headerContainerStyle={styles.tabBarStyle}>
		{tabs.map((tab) => {
			switch (tab.key) {
				case 'about':
					return (
						<Tabs.Tab name={tab.title} key={'about'}>
							<AboutTab />
						</Tabs.Tab>
					);
				case 'products':
					return (
						<Tabs.Tab name={tab.title} key={'products'}>
							<ProductsTab translationY={translationY} />
						</Tabs.Tab>
					);
				case 'categories':
					return (
						<Tabs.Tab name={tab.title} key={'categories'}>
							<CategoriesTab />
						</Tabs.Tab>
					);
				case 'reviews':
					return (
						<Tabs.Tab name={tab.title} key={'reviews'}>
							<ReviewsTab />
						</Tabs.Tab>
					);
				default:
					return null;
			}
		})}
	</Tabs.Container>
</View>
);
```

![123](https://github.com/PedroBern/react-native-collapsible-tab-view/assets/99968085/8e1a5e76-6fdd-4430-aada-4426fc936964)

**Comment by davram88** [comment]
Same problem, any update?

**Comment by devoren** [comment]
@davram88 nope :(

**Comment by davram88** [comment]
It seems that whitespace is based on the height of the header. If you remove the header completely from the container you won't have that space anymore. Still no final solution tho :(

**Comment by davram88** [comment]
I added cancelLazyFadeIn to my Tabs.Lazy and now I don't have that initial white space when I switch tabs.

`    <Tabs.Container ref={ref} pagerProps={{scrollEnabled: false}} key={elementProps.keyStr}
                        minHeaderHeight={minHeaderHeight}
                        renderHeader={() => elementProps.header}
                        renderTabBar={tabBar}>
          
          {elementProps.tabs.map(tab => (
            <Tabs.Tab name={tab.name}>
              <Tabs.Lazy key={tab.name} cancelLazyFadeIn>
                <Tabs.ScrollView>
                  {tab.view}
                </Tabs.ScrollView>
              </Tabs.Lazy>
            </Tabs.Tab>
          ))}
        </Tabs.Container>`

**Comment by devoren** [comment]
@davram88 oh okay i will try it thank you

**Comment by ivanichoo** [comment]
Hello,

Got the same bug with Tabs.FlashList and dynamic height. I have been trying all solutions above but nothing works or is stable.

"expo": "~48.0.12",
"react-native": "0.71.8",
"react-native-collapsible-tab-view": "^6.2.0"

Thank you!

**Comment by devoren** [comment]
Guys, in my case, the name of the tabs was in a specific language, and not in English, because of this, such a problem arose. @davram88, @ivanichoo if your tab name is in another language try english and use the label as the tab name. So I'm closing the issue as it's been resolved

**Comment by miguelespinoza** [comment]
I seem to have the same issue but with Tabs.FlashList and lazy, I created a new issue here
https://github.com/PedroBern/react-native-collapsible-tab-view/issues/373

**Comment by metinaltinbas** [comment]
@devoren I don't think it is related to the language issue, some of others also having the same problem with english name and labels.

**Comment by devoren** [comment]
@metinaltinbas yeah i saw :(

**Comment by miladdev85** [comment]
I have this issue with ```Tabs.FlatList```. Any solution?

Using:
```
"react-native-collapsible-tab-view": "^5.0.0-rc.13",
"react-native-pager-view": "^5.4.25",
"react-native-reanimated": "2.14.4",
```

**Comment by devoren** [comment]
@miladdev85 wow really?! i closed and now i  see bug again (and it doesn't matter what list it is)

**Comment by miladdev85** [comment]
> @miladdev85 wow really?! i closed and now i  see bug again (and it doesn't matter what list it is)
> 
> 

Yes, it's gone for me. Only appears if I debug with Chrome. Haven't tried out TestFlight build yet... 😬

**Comment by metinaltinbas** [comment]
I still have the bug with FlashList and lazy

**Comment by iy-913** [comment]
I encountered the same problem in a multilingual application and it turned out that for the English language everything works correctly, but with other languages ​​there is this problem.

Before my code was like

<Tabs.Tab name={'Some tab name'}>

After my code is like

<Tabs.Tab name={'Some tab name ID'} label={'Some tab label text'}>

Where label is displayed text and everything works fine.

**Comment by webdobe** [comment]
I came across this issue as well.  Here is what it looks like some key values are setting the "estimated" and the disableHorizontalListHeightMeasurement values:

```
<Tabs.Lazy startMounted={false} cancelLazyFadeIn>
      {posts && posts.length ? (
        <Tabs.MasonryFlashList
          disableHorizontalListHeightMeasurement
          data={posts}
          contentContainerStyle={{padding: 10}}
          numColumns={3}
          estimatedItemSize={200}
          estimatedListSize={{width: Dimensions.get("window").width, height: Dimensions.get("window").height}}
          keyExtractor={(item) => item.id}
          estimatedFirstItemOffset={0}
          renderItem={({item}) => {
            const media = item?.getValue("field_media");
            const url = getMediaUrl(media, "large");
            return (
              // Inserted component here. 
            )
          }}
          onEndReached={onEndReached}
          onEndReachedThreshold={0.5}
          onRefresh={onRefresh}
          refreshing={isLoading}
        />
      ) : null}
    </Tabs.Lazy>

```

**Comment by ShashankSai2110** [comment]
I am also getting the same issue. Can anyone guide to resolve this

**Comment by AbdoHema2016** [comment]
here is the [fix](https://github.com/PedroBern/react-native-collapsible-tab-view/issues/259#issuecomment-1948019659) I came up with

**Comment by VictorioMolina** [comment]
Any updates?

**Comment by andreialecu** [comment]
Is this still an issue with version 6.2.2?

Can someone make a repro on top of the Example app in the repo?

**Comment by andreialecu** [comment]
https://github.com/PedroBern/react-native-collapsible-tab-view/pull/394 might fix this, I released it as 7.0.1-beta.0 on npm. Please help test it for unwanted side effects and report back. Thanks!

**Comment by hikitty2** [comment]
> #394 might fix this, I released it as 7.0.1-beta.0 on npm. Please help test it for unwanted side effects and report back. Thanks!#394 可能会解决这个问题，我在 npm 上将其发布为 7.0.1-beta.0。请帮助测试它是否有不需要的副作用并报告。谢谢！

The version 7.0.1-beta.0 still has the same issue.

**Comment by namdang0219** [comment]
<Tabs.Tab name="tags" label='タグ'>
use english for name and different language for label props, this bug will gone.
it work for me 🌟
